images/rootfs.yml.in

```diff
@@ -11,10 +11,13 @@ init:
   - linuxkit/containerd:417f83f7b8dc1fa36acf90effe44f99c7397480a
```

Comment:
Given the upcoming demo, don't change rootfs.yml. This will have no effect as of now, as it won't be used by any image in this commit.